lib/puppet/type/docker_compose.rb

```diff
@@ -38,6 +38,13 @@ def refresh
 		end
 	end
 
+	newparam(:restart) do
```

Comment:
I think this should have a default value of true. This would make it optional and preserve backwards compatibility I think?